### PR TITLE
set plugin/device for run_infer script used in CI

### PR DESCRIPTION
**Pull request by bani-intelaipg**

### CODE REVIEW

**Review by adk9 — CHANGES_REQUESTED** [review]
LGTM. Please comment out the failing models in `models-myriad` pipeline.

**Review by adk9 — APPROVED** [review]

File: test/ci/buildkite/models-cpu.yml [hunk 1]
```diff
@@ -16,9 +16,12 @@
   - wait
   - command: |
       source /localdisk/buildkite/artifacts/$BUILDKITE_BUILD_ID/venv/bin/activate
+      device=${NGRAPH_TF_BACKEND:-"CPU"}
+      device=$(echo "\$device" | tr '[:upper:]' '[:lower:]')
+      MODELFILENAME=models_\${device}.txt
```
Comment (adk9) [inline test/ci/buildkite/models-cpu.yml:21]: Not needed since run_infer_multiple.sh knows how to use the correct device manifest?

Comment (bani-intelaipg) [reply test/ci/buildkite/models-cpu.yml:21]: In local dev/testing, we might want to pass a custom manifest to the run_infer_multiple.sh for targeted testing - so need a way to specify the param

Comment (adk9) [reply test/ci/buildkite/models-cpu.yml:21]: Sure, but the correct one will be used if one isn't specified? Why do you need to explictly specify a manifest here?

Comment (adk9) [reply test/ci/buildkite/models-cpu.yml:21]: I'm marking this as resolved in the interest of merging this PR ASAP to enable testing of other outstanding PRs.

Please remove the explicit setting of model manifest above in a follow-up PR.

**Pull request merged**